payments/credit-card/direct.md

```diff
@@ -1,194 +0,0 @@
----
```

Comment:
`direct.md` is detected as deleted, probably because the rename operation was not committed separately from the large change of the contents of the file. For Git to detect file renames, the file needs to match in content above 60%. It is therefore best to commit file renames separately from file changes. We can try to fix this together.